content/en/faq/general/foundation.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Is Istio a member project of a foundation like the Linux Foundation, CNCF or OASIS?
+weight: 90
+---
+
+No, Istio is an open source project under the [Apache License 2.0]({{< github_blob >}}/LICENSE) that exercises an
+[open governance model](https://github.com/istio/community) with a multi-company [steering committee](https://github.com/istio/community/blob/master/STEERING-COMMITTEE.md),
+[technical oversight committee](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md) and distributed [working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+There are no plans to donate Istio to a foundation at this time.
```

Comment:
can we remove this line about plans?

Reply:
OK, it's gone. You OK with checking this in?

Reply:
The IBM team would like to have us wait on publishing this FAQ till the senior team have synced and agreed to this, @geeknoid @jdumars